addons/mail/static/tests/mobile/mobile.test.js

```diff
@@ -64,3 +66,17 @@ test("can leave channel in mobile", async () => {
     await click(".o-mail-ChatWindow-command", { text: "General" });
     await contains(".o-dropdown-item", { text: "Leave" });
 });
+
+test.tags("mobile");
```

Comment:
Wasn't there an issue with tag mobile on a file with a global tag desktop?

Reply:
Good question, I will look into it quickly now

Reply:
I change the tag of all tests in file from "desktop" to "mobile"